client/src/components/Workflow/Editor/Forms/FormSection.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+    <Form :id="id" :inputs="inputs" ref="form" @onChange="onChange" />
+</template>
+
+<script>
+import Form from "components/Form/Form";
+import makeSection from "./makeSection";
+
+export default {
+    components: {
+        Form,
+    },
+    props: {
+        id: {
+            type: String,
+            required: true,
+        },
+        getNode: {
+            type: Function,
+            required: true,
+        },
```

Comment:
I think I need to understand how you're setting up your node tree hierarchy.

Passing funcs in like this is usually problematic. Is there a reason the output of the getNode() function couldn't be passed in directly as a "node" property?

Reply:
`Node` is Vue component, as of right now some of this components functions are still called directly and parsing the entire `Node` as reactive proved to produce unexpected issues. By parsing a static function we make sure that there is no reactive overspill potentially leading to recursions. As mentioned above this was of major utility when transitioning the legacy workflow code. Also it helps to tag these for the future, so that we can successively remove the parsed prop functions one component at a time.

Reply:
If you find yourself passing in a component as a prop even through a functional retrieval like this, you should probably be looking into slots as a better paradigm for nesting a dynamic child component from upstream.